src/level_four/core/tasks/send_transaction_tasks.c

```diff
@@ -139,13 +133,12 @@ void send_transaction_tasks()
         char display[225] = {0};
         uint8_t precision = get_floating_precision(value, SATOSHI_PER_BTC);
         snprintf(display, sizeof(display), "Receiver #%d\nSend %0.*f\n%s", var_send_transaction_data.transaction_confirmation_list_index + 1, precision, valueToDisplay, get_coin_symbol(BYTE_ARRAY_TO_UINT32(var_send_transaction_data.transaction_metadata.coin_index), var_send_transaction_data.transaction_metadata.network_chain_id));
-        confirm_scr_init(display);
+        ui_scrollabe_page(ui_heading_confirm_action, display, MENU_SCROLL_HORIZONTAL, false);
```

Comment:
Spelling mistake causing errors

Reply:
did not push all changes before starting this review.
fixed. pls re-review